apps/camera/js/app.js

```diff
@@ -363,7 +363,9 @@ App.prototype.onBeforeUnload = function() {
  * @private
  */
 App.prototype.loadL10n = function(done) {
-  this.require(['l10n'], done);
+  this.require(['l10n'], function() {
+    this.require(['l10n_date'], done);
+  });
```

Comment:
I haven't worked with require.js based code much, so I don't know about this kind of nested require. Can you pass l10n and l10n_date in the same array?  I think you can probably add something to js/config/require.js to specify that l10n_date requires l10n so that it should be safe to load them in parallel.

My concern here is that nesting these two will have an impact on app startup time.

I think you'll need to ask Wilson Page or Diego Marcos to review these changes.